corehq/apps/app_manager/views/releases.py

```diff
@@ -103,6 +103,7 @@ def paginate_releases(request, domain, app_id):
     for app in saved_apps:
         app['include_media'] = app['doc_type'] != 'RemoteApp'
         app['j2me_enabled'] = app['menu_item_label'] in j2me_enabled_configs
+        app['target_commcare_flavor'] = SavedAppBuild.get(app['_id']).target_commcare_flavor
```

Comment:
This is going to fetch each saved app individually from couch, right? Do you think it would be worth skipping this if the toggle is off?

Reply:
I think the best fix would be for the ```app_manager/saved_app``` view to include ```target_commcare_flavor```.  Right now it seems to be omitting that field.

Reply:
Although, I suppose if you did turn off the toggle, it would also be ok to just pretend that ```target_commcare_flavor``` is always ```'none'```.  That sounds like the right behavior irregardless of the performance optimization of avoiding another CouchDB hit.

Reply:
https://github.com/dimagi/commcare-hq/pull/20233

Reply:
@nickpell Does this mean that in `profile.ccpr` file can have `target_commcare_flavor`  as "none".  If that's the case I will need to factor that in mobile code.

Reply:
No, if ```target_commcare_flavor``` is ```'none'``` then the extra line in profile.ccpr will be omitted.

Reply:
👍